notebooks/tutorial_RGDR.ipynb

```diff
@@ -70,7 +70,7 @@
    "outputs": [
```

Comment:
<p>To be complete, also add <code>rgdr.pval_map[0].plot()</code>, preferably in the same cell parallel to the correlation map.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI4S2S/s2spy/pull/94/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
New cell added in the notebook to show the `rgdr.pval_map[0].plot()` now.